examples/demo_vxlan.pp

```diff
@@ -25,17 +25,28 @@
     dup_host_mac_detection_timeout        => '10',
   }
 
```

Comment:
Given that we currently only support `vxlan` on n9k and possibly n3k, lets use the `platform_get()` api in `demo_all.pp` to only incude `demo_vlan.pp` on supported platforms.

```
...
  include ciscopuppet::demo_vtp
  include ciscopuppet::demo_snmp
  # Conditionally include ciscopuppet::demo_vxlan
  if platform_get() =~ /n(3|9)k/ {
    include ciscopuppet::demo_vxlan
  }
}
```
Puppet Reference: https://docs.puppetlabs.com/puppet/latest/reference/lang_conditional.html#if-statements